### PR TITLE
RD-3672 Edit Mode spec adjustments

### PR DESCRIPTION
**Pull request by qooban**
## Description

As we all know Edit Mode spec is quite flaky. The flakiness comes from verification of `POST /console/ua` calls response body.
I think we can live without those checks without losing much, so I removed all those assertions and removed test retries as they were not helping to solve the flakiness.

Ref. RD-3672

## Screenshots / Videos
N/A

## Checklist
- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests
1. [Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/2329/) - only Edit Mode spec - PASSED
2. [Build Status](https://jenkins.cloudify.co/job/Stage-UI-System-Test/2330/) - only Edit Mode spec - PASSED

## Documentation
N/A

### CODE REVIEW

File: test/cypress/integration/edit_mode_spec.ts [hunk 1]
```diff
@@ -267,12 +252,13 @@ describe('Edit mode', () => {
             const widgetName = 'Backend';
             const widgetId = `testWidget${widgetName}`;
             const url = 'https://this-page-intentionally-left-blank.org/';
-            const verifyRequest = service =>
+            const verifyRequest = (service: string) =>
                 cy.wait(service).then(({ request, response }) => {
                     expect(request.headers).contain({ 'widget-id': widgetId });
-                    expect(response.statusCode).equals(200);
+                    expect(response!.statusCode).equals(200);
```
Comment (kubama) [inline test/cypress/integration/edit_mode_spec.ts:258]: I think it is always cleaner:
```suggestion
                    expect(response?.statusCode).equals(200);
```

**Review by kubama — COMMENTED** [review]
As I understand removing the checks does not guarantee that the issue is fixed. Maybe it would be good to just comment out the checks and provide additional comment(s) explaining why the checks are disabled? In case somebody wanted to again introduce missing checks, but also in case it turned out that checks are not the issue but the actual endpoint response. It's also possible that the root cause of the issue is identified/fixed in the future, so it will be possible to re-enable the checks easily.

**Review by NorbertAmartus — APPROVED** [review]

**Comment by qooban** [comment]
@kubama 
> As I understand removing the checks does not guarantee that the issue is fixed. Maybe it would be good to just comment out the checks and provide additional comment(s) explaining why the checks are disabled? In case somebody wanted to again introduce missing checks, but also in case it turned out that checks are not the issue but the actual endpoint response. It's also possible that the root cause of the issue is identified/fixed in the future, so it will be possible to re-enable the checks easily.

Hmm... I'm not sure. I have compared videos/screenshots for passing and failed tests I don't see a way how could it be possible that Edit Mode spec will fail in the same places without response body verification. I can revert checks and comment it out, but will that be valuable for us? We have Git history.

**Comment by kubama** [comment]
> Hmm... I'm not sure. I have compared videos/screenshots for passing and failed tests I don't see a way how could it be possible that Edit Mode spec will fail in the same places without response body verification.

My thinking is that it can still fail in subsequent places if it's an issue with actual endpoint response and not just the verification.

> I can revert checks and comment it out, but will that be valuable for us? We have Git history.

Having it commented out with additional comment explaining the reason will keep the awareness of the issue, which is beneficial for the reasons I mentioned in my previous comment.

This is optional comment, so I'm approving regardless of your final decision.

**Comment by qooban** [comment]
> My thinking is that it can still fail in subsequent places if it's an issue with actual endpoint response and not just the verification.

From what I have investigated that's very unlikely, 'cause it seems like the proper request is sent anyway. The problem is - as I suspect - that sometimes it is being sent twice (there are 2 updates instead of 1). But all in all the result in the UI view is OK.

> This is optional comment, so I'm approving regardless of your final decision.

Thanks. I'll keep it as is for now.

**Pull request merged**